### DISCUSSION TITLE
DoPolarizationSums gives differen results for different auxiliary vectors

### DISCUSSION THREAD

**Discussion by FloHech** [FeynCalc/feyncalc#211] [Q&A]
Hi,
I am encountering a peculiarity with DoPolarizationSums that I do not understand. The problem under consideration is a three body decay of the form 
G(p1)->\pi(p2)+\pi(p3)+\gamma(p4),
where all particles are scalar or pseudoscalar and massive, except the photon.

Suppose I have the amplitude
(I*d5*(Pair[Momentum[p1], Momentum[p2]] - Pair[Momentum[p1], Momentum[p3]])*Pair[Momentum[p1], Momentum[Polarization[p4, -I, Transversality -> True]]])/m^2 + 
 I*d5*(Pair[Momentum[p2], Momentum[Polarization[p4, -I, Transversality -> True]]] - Pair[Momentum[p3], Momentum[Polarization[p4, -I, Transversality -> True]]])

When I then compute the squared matrix element via DoPolarizationSums[ComplexConjugate[%] % ,p4 , n] I get different results depending on which auxiliary vector I choose e.g. n= p1, p2 or p3.  When the result should of course be independent of that choice.

The scalar product for p4 is set on-shell. The Lagrangian looks like this (or the d5 term in (B5) of 1501.07906):
QuantumField[GaugeField, LorentzIndex[\[Nu]1]]*(QuantumField[\[CapitalPi]2]*QuantumField[FCPartialD[LorentzIndex[\[Mu]1]], \[CapitalPi]1] - 
  QuantumField[\[CapitalPi]1]*QuantumField[FCPartialD[LorentzIndex[\[Mu]1]], \[CapitalPi]2])*(MT[\[Mu]1, \[Nu]1]*QuantumField[Gd] - 
  QuantumField[FCPartialD[LorentzIndex[\[Mu]1]], FCPartialD[LorentzIndex[\[Nu]1]], Gd]/m1SS^2)

I also encountered the problem in a more complicated decay of a pseudovector into a pseudoscalar + vector + photon. But the problem is probably the same in both cases. So far I had no difficulties with this method for two-body decays even when the interactions are coming from a Chern-Simons term, which is not gauge invariant. 

Any help or comment is much appreciated!
Thanks in advance!

**Comment by vsht** [comment]
Hi,

`DoPolarizationSums` just substitutes the standard text-book formula for the polarization sum involving an auxiliary vector, 
so there is hardly anything special about the function itself. You can do essentially the same by uncontracting polarization
vectors with `Uncontract` and plugging any polarization sum you like.

I'd say that either the difference between expressions involving different `n` values is indeed zero (but one doesn't see it immediately)
or there is an issue with your amplitude/model. As you said yourself, the results normally should not differ for a gauge invariant theory.

If you suspect that the problem is on the FeynCalc side, you could do a cross-check with FormCalc or CalcHEP to be safe.

Cheers,
Vladyslav
